pkg/security/probe/syscall_stats.go

```diff
@@ -160,8 +225,25 @@ func NewSyscallMonitor(manager *manager.Manager) (*SyscallMonitor, error) {
 		return nil, errors.New("map noisy_processes_bb not found")
 	}
 
+	execFrontBuffer, ok, err := manager.GetMap("exec_count_fb")
+	if err != nil {
+		return nil, err
+	}
+	if !ok {
+		return nil, errors.New("map exec_count_one not found")
+	}
```

Comment:
Would it make sense to update `func (m *Manager) GetMap(name string)` to make it return only an `*ebpf.Map` and an `error` and to have it return a dedicated error type in case of map not found ?

Because it seems that in most of the `manager.GetMap(…)` invocation, we end up:
* forwarding the error in case of error and;
* building a new error in case of map not found.

So, it could be interesting to move the new error building directly inside `manager.GetMap(…)`.

Reply:
That's a good point. However this requires an update of the library and a change of all the invocations of `manager.GetMap()` in system-probe. We should probably handle this in a separate PR. I'll go ahead and close this thread, but re-open if needed.